### PR TITLE
include filename/url in clusterIO timeout/error logging

### PR DESCRIPTION
**Pull request by barentine**
Addresses issue # .

**Is this a bugfix or an enhancement?**
enhancement
**Proposed changes:**
- include filename/url in logging statements

**Checklist:**

- [ ] Tested with numpy=1.14
- [ ] Tested on python 2.7 and 3.6
- [ ] Tested with wx=3.x and wx=4.x [if UI code]
- [x] Does the PR avoid variable renaming in existing code, whitespace changes, and other forms of tidying? [There is a place for code tidying, but it makes reviewing 
much simpler if this is kept separate from functional changes]

If an enhancement (or non-trivial bugfix):

- [ ] Has this been discussed in advance (feature request, PR proposal, email, or direct conversation)?
- [ ] Does this change how users interact with the software? How will these changes be communicated?
- [x] Does this maintain backwards compatibility with old data?
- [ ] Does this change the required dependencies?
- [ ] Are there any other side effects of the change?

### CODE REVIEW

**Comment by barentine** [comment]
as an aside, I'll update the nodes to the repo head and try again, but it seems like we may have nerf'd something on the dataservers - trying to download a 30k frame series, or even just take a slice through the stack opened in dh5view, is like pulling teeth at the moment, with many retries starting as early as frame 600 and then eventually exceeding the 3 retry count and failing on frames which in fact exist

File: PYME/IO/clusterIO.py [hunk 1]
```diff
@@ -803,10 +803,10 @@ def get_file(filename, serverfilter=local_serverfilter, numRetries=3, use_file_c
         except (requests.Timeout, requests.ConnectionError) as e:
             # s.get sometimes raises ConnectionError instead of ReadTimeoutError
             # see https://github.com/requests/requests/issues/2392
-            logger.exception('Timeout on get file')
+            logger.exception('Timeout on get file %s' % url)
             logger.info('%d retries left' % (numRetries - nTries))
             if nTries == numRetries:
-                raise
+                raise e
```
Comment (David-Baddeley) [inline PYME/IO/clusterIO.py]: is this strictly needed? I thought that `raise` on it's own in an exception block just re-raised the exception that was being handled (i.e. didn't need the explicit exception argument), and was more idiomatic python.

Comment (barentine) [reply PYME/IO/clusterIO.py]: Not necessary, nor idiomatic. Same could be said for assigning the exception to an unused variable on 803. Would you like me to revert it?

Comment (David-Baddeley) [reply PYME/IO/clusterIO.py]: was more a  "wait, did I have this wrong" informational question. Suspect the `except as e` bit was left over from looking at some additional info in the exception object.

Comment (barentine) [reply PYME/IO/clusterIO.py]: Well, it's on me for changing something I didn't need to, sorry. Went ahead and took out the `as e`.

**Pull request merged**